### PR TITLE
Add demo for ViewItemDelegate class added to tk-framework-qtwidgets

### PR DESCRIPTION
**Pull request by staceyoue**

### CODE REVIEW

File: python/tk_multi_demo/demos/view_item_delegate/ui/resources_rc.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+qt_resource_name = b"\x00 \x0e\xdeh\x05\x00t\x00k\x00_\x00m\x00u\x00l\x00t\x00i\x00_\x00d\x00e\x00m\x00o\x00_\x00v\x00i\x00e\x00w\x00_\x00i\x00t\x00e\x00m\x00_\x00d\x00e\x00l\x00e\x00g\x00a\x00t\x00e\x00\x0d\x09\x10A\xc7\x00p\x00r\x00o\x00j\x00e\x00c\x00t\x00_\x001\x00.\x00p\x00n\x00g\x00\x0e\x04\xa2\xfc\xa7\x00d\x00o\x00w\x00n\x00_\x00a\x00r\x00r\x00o\x00w\x00.\x00p\x00n\x00g\x00\x0d\x09\x09A\xc7\x00p\x00r\x00o\x00j\x00e\x00c\x00t\x00_\x002\x00.\x00p\x00n\x00g"
+qt_resource_struct = b"\x00\x00\x00\x00\x00\x02\x00\x00\x00\x01\x00\x00\x00\x01\x00\x00\x00\x00\x00\x02\x00\x00\x00\x03\x00\x00\x00\x02\x00\x00\x00f\x00\x00\x00\x00\x00\x01\x00\x00c\xef\x00\x00\x00\x88\x00\x00\x00\x00\x00\x01\x00\x00eK\x00\x00\x00F\x00\x00\x00\x00\x00\x01\x00\x00\x00\x00"
+def qInitResources():
+    QtCore.qRegisterResourceData(0x01, qt_resource_struct, qt_resource_name, qt_resource_data)
+
+def qCleanupResources():
+    QtCore.qUnregisterResourceData(0x01, qt_resource_struct, qt_resource_name, qt_resource_data)
+
+qInitResources()
```
[the remark this answers is not in the document]
Comment (staceyoue) [reply python/tk_multi_demo/demos/view_item_delegate/ui/resources_rc.py:20]: This file is autogenerated

File: python/tk_multi_demo/demos/view_item_delegate/ui/resources_rc.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+qt_resource_name = b"\x00 \x0e\xdeh\x05\x00t\x00k\x00_\x00m\x00u\x00l\x00t\x00i\x00_\x00d\x00e\x00m\x00o\x00_\x00v\x00i\x00e\x00w\x00_\x00i\x00t\x00e\x00m\x00_\x00d\x00e\x00l\x00e\x00g\x00a\x00t\x00e\x00\x0d\x09\x10A\xc7\x00p\x00r\x00o\x00j\x00e\x00c\x00t\x00_\x001\x00.\x00p\x00n\x00g\x00\x0e\x04\xa2\xfc\xa7\x00d\x00o\x00w\x00n\x00_\x00a\x00r\x00r\x00o\x00w\x00.\x00p\x00n\x00g\x00\x0d\x09\x09A\xc7\x00p\x00r\x00o\x00j\x00e\x00c\x00t\x00_\x002\x00.\x00p\x00n\x00g"
+qt_resource_struct = b"\x00\x00\x00\x00\x00\x02\x00\x00\x00\x01\x00\x00\x00\x01\x00\x00\x00\x00\x00\x02\x00\x00\x00\x03\x00\x00\x00\x02\x00\x00\x00f\x00\x00\x00\x00\x00\x01\x00\x00c\xef\x00\x00\x00\x88\x00\x00\x00\x00\x00\x01\x00\x00eK\x00\x00\x00F\x00\x00\x00\x00\x00\x01\x00\x00\x00\x00"
+def qInitResources():
+    QtCore.qRegisterResourceData(0x01, qt_resource_struct, qt_resource_name, qt_resource_data)
+
+def qCleanupResources():
```
[the remark this answers is not in the document]
Comment (staceyoue) [reply python/tk_multi_demo/demos/view_item_delegate/ui/resources_rc.py:17]: Ignoring -- this file is autogenerated

File: python/tk_multi_demo/demos/view_item_delegate/ui/resources_rc.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+qt_resource_name = b"\x00 \x0e\xdeh\x05\x00t\x00k\x00_\x00m\x00u\x00l\x00t\x00i\x00_\x00d\x00e\x00m\x00o\x00_\x00v\x00i\x00e\x00w\x00_\x00i\x00t\x00e\x00m\x00_\x00d\x00e\x00l\x00e\x00g\x00a\x00t\x00e\x00\x0d\x09\x10A\xc7\x00p\x00r\x00o\x00j\x00e\x00c\x00t\x00_\x001\x00.\x00p\x00n\x00g\x00\x0e\x04\xa2\xfc\xa7\x00d\x00o\x00w\x00n\x00_\x00a\x00r\x00r\x00o\x00w\x00.\x00p\x00n\x00g\x00\x0d\x09\x09A\xc7\x00p\x00r\x00o\x00j\x00e\x00c\x00t\x00_\x002\x00.\x00p\x00n\x00g"
+qt_resource_struct = b"\x00\x00\x00\x00\x00\x02\x00\x00\x00\x01\x00\x00\x00\x01\x00\x00\x00\x00\x00\x02\x00\x00\x00\x03\x00\x00\x00\x02\x00\x00\x00f\x00\x00\x00\x00\x00\x01\x00\x00c\xef\x00\x00\x00\x88\x00\x00\x00\x00\x00\x01\x00\x00eK\x00\x00\x00F\x00\x00\x00\x00\x00\x01\x00\x00\x00\x00"
+def qInitResources():
```
[the remark this answers is not in the document]
Comment (staceyoue) [reply python/tk_multi_demo/demos/view_item_delegate/ui/resources_rc.py:14]: Ignoring -- this file is autogenerated

File: python/tk_multi_demo/demos/view_item_delegate/__init__.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Autodesk, Inc.
+
+from .demo import ViewItemDelegateDemo
```
[the remark this answers is not in the document]
Comment (staceyoue) [reply python/tk_multi_demo/demos/view_item_delegate/__init__.py:11]: We need this for the demo to be discoverable

**Review by mathurf — CHANGES_REQUESTED** [review]
Please fix code style validation issues (Black) by running pre-commit locally

**Review by mathurf — APPROVED** [review]
LGTM

**Review by 000paradox000 — APPROVED** [review]

**Pull request merged**